suite-native/graph/src/components/Graph.tsx

```diff
@@ -1,19 +1,29 @@
-import React, { useMemo } from 'react';
+import React, { useCallback, useMemo, useRef, useState } from 'react';
 
-import { N } from '@mobily/ts-belt';
+import { G, N } from '@mobily/ts-belt';
 
-import { GraphPoint, LineGraph } from '@suite-native/react-native-graph';
+import { GraphEvent, GraphPoint, LineGraph } from '@suite-native/react-native-graph';
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 import { Box, Loader } from '@suite-native/atoms';
-
 import {
-    getExtremaFromGraphPoints,
-    EnhancedGraphPointWithCryptoBalance,
-    EnhancedGraphPoint,
-} from '../utils';
+    FiatGraphPoint,
+    FiatGraphPointWithCryptoBalance,
+    GroupedBalanceMovement,
+} from '@suite-common/graph';
+
+import { getExtremaFromGraphPoints } from '../utils';
 import { AxisLabel, MAX_CLAMP_VALUE } from './AxisLabel';
 import { GraphError } from './GraphError';
+import { TransactionEventTooltip, TransactionEventTooltipProps } from './TransactionEventTooltip';
 import { SelectionDotWithLine } from './SelectionDotWithLine';
+import { TransactionEvent } from './TransactionEvent';
+
+type GraphTransactionEventPayload = GroupedBalanceMovement['payload'] & {
+    handleActiveTransactionEvent: (
```

Comment:
This is very atypical API for this purpose, usually you have prop like `onEventPress` for Graph component and that is passed internally to Event component as prop.

Reply:
Fixed in 90955f4fc by moving the tooltip inside of the `react-native-graph` library.